testapp/tests.py

```diff
@@ -0,0 +1,91 @@
+from collections import defaultdict
+from collections.abc import Iterable
+from math import log, sqrt
+
+import pytest
+from django.db.models import Count
+from django.db.models.functions import TruncDate
+from django_pg_simple_hll.aggregate import ApproxCardinality
+
+from .models import Session
+
+FIELDS = ("user_int", "user_uuid", "user_str")
+PRECISIONS_TO_TEST = (5, 8, 9, 10, 11, 12)  # if tails intermittently with 4
```

Comment:
```suggestion
PRECISIONS_TO_TEST = (5, 8, 9, 10, 11, 12)  # it fails intermittently with 4
```

Does this mean we're not testing the `m=16` and `m=64` cases of the alpha_m approximation constant?

If we're getting intermittent failures do we just need to re-run the test more times to get a better average of averages?

Reply:
added back here: 060374d5cb36abde59482ff8929302dc6442a3aa